### PR TITLE
feat: custom sap ODM extensions

### PR DESCRIPTION
**Pull request by aramovic79**

### CODE REVIEW

**Comment by aramovic79** [comment]
@pavelkornev : Could you please review? Thanks.

**Comment by pavelkornev** [comment]
@aramovic79 the PR looks good to me. Thank you for the contribution. We will merge it once all alignments are through.

**Comment by aramovic79** [comment]
> @aramovic79 the PR looks good to me. Thank you for the contribution. We will merge it once all alignments are through.

👍

**Comment by aramovic79** [comment]
> @aramovic79 the PR looks good to me. Thank you for the contribution. We will merge it once all alignments are through.

Latest changes from https://github.tools.sap/CentralEngineering/sap-openapi-specification/pull/48 included.
@pavelkornev : Could you please review again and let me know if should merge this PR after the 48?

**Review by pavelkornev — APPROVED** [review]

**Review by swennemers — DISMISSED** [review]

**Review by ralfhandl — COMMENTED** [review]
Synchronize descriptions in v2 and v3 schemas?

File: sap-extensions/v2.0-extensions.json [hunk 1]
```diff
@@ -3,6 +3,19 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "x-extends": [
+    {
+      "paths": ["definitions.schema.properties"],
+      "properties": {
+        "x-sap-odm-entity-name": {
+          "type": "string",
+          "description": "Name of an ODM entity as a general concept, not a concrete version thereof"
```
[the remark this answers is not in the document]
Comment (aramovic79) [reply sap-extensions/v2.0-extensions.json:11]: @swennemers @pavelkornev : The description should be the same as in v3.0, right?

[the remark this answers is not in the document]
Comment (pavelkornev) [reply sap-extensions/v2.0-extensions.json:11]: Those are your fields, you should know it :) From my POV I agree with Ralf that there is no reason why descriptions should be any different between two versions of the spec.

[the remark this answers is not in the document]
Comment (swennemers) [reply sap-extensions/v2.0-extensions.json]: Thanks @ralfhandl for catching this, yes, there's no reason to have them differently. @aramovic79 can you align them please?

**Review by pavelkornev — APPROVED** [review]

**Pull request merged**